src/Gzero/Cms/Http/Controllers/Api/ContentFileController.php

```diff
@@ -0,0 +1,174 @@
+<?php namespace Gzero\Cms\Http\Controllers\Api;
+
+use Gzero\Cms\Repositories\ContentReadRepository;
+use Gzero\Cms\Validators\ContentValidator;
+use Gzero\Core\Http\Controllers\ApiController;
+use Gzero\Core\Http\Resources\FileCollection;
+use Gzero\Core\Jobs\SyncFiles;
+use Gzero\Core\Models\File;
+use Gzero\Core\UrlParamsProcessor;
+use Illuminate\Http\Request;
+
+/**
+ * Class ContentFileController
+ *
+ * @SWG\Tag(
+ *   name="content",
+ *   description="Everything about app content"
```

Comment:
This swagger tag already exists, so there will be an error during docs generation. Please make sure that swagger docs are generating with your new methods.

Reply:
There is no error, I checked it. We can leave it as that or remove this doc block completely like it is done in `ContentTranslationController`.